docs/resources/secret.md

```diff
@@ -0,0 +1,55 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "konnect_secret Resource - terraform-provider-konnect-beta"
```

Comment:
@slonka apologies for missing this before - but is this resource only relevant for mesh?
In that case would it make sense to call it something like `konnect_mesh_secret`?
Same for zone_egress and zone_ingress

Reply:
@harshadixit12 - that's not the case, not all are mesh prefixed - e.g. https://github.com/Kong/terraform-provider-konnect-beta/blob/main/openapi.yaml#L19420